components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -478,8 +478,12 @@ private String getAuthzUser(AccessTokenDO accessTokenDO) throws IdentityOAuth2Ex
 
         String consumerKey = accessTokenDO.getConsumerKey();
         try {
-            boolean buildSubjectIdentifierFromSPConfig = Boolean.parseBoolean(IdentityUtil.getProperty
-                    (BUILD_FQU_FROM_SP_CONFIG));
+            boolean buildSubjectIdentifierFromSPConfig = true;
+            if (StringUtils.isNotEmpty(IdentityUtil.getProperty(BUILD_FQU_FROM_SP_CONFIG))) {
```

Comment:
Shall we assign a variable to **IdentityUtil.getProperty(BUILD_FQU_FROM_SP_CONFIG** and reuse it?